tox.ini

```diff
@@ -1,80 +1,79 @@
 [tox]
 envlist = flake8, pylint, check-{py27,py36}, coveralls
-skipsdist = true
-skip_missing_interpreters = True
```

Comment:
On platforms without python3, I think the run will not work.

Reply:
I'm not 100% sure, but from what I understood , "interpreter" here is a tox environment. Will try to run it on Python3-less system.